src/main/java/ch/njol/skript/effects/EffBroadcast.java

```diff
@@ -31,29 +36,32 @@
 import ch.njol.skript.lang.VariableString;
 import ch.njol.skript.registrations.Classes;
 import ch.njol.skript.util.LiteralUtils;
+import ch.njol.skript.util.SkriptColor;
+import ch.njol.skript.util.Utils;
 import ch.njol.skript.util.chat.BungeeConverter;
 import ch.njol.skript.util.chat.ChatMessages;
 import ch.njol.util.Kleenean;
+import ch.njol.util.StringUtils;
 import ch.njol.util.coll.CollectionUtils;
 import net.md_5.bungee.api.chat.BaseComponent;
 import org.bukkit.Bukkit;
 import org.bukkit.World;
 import org.bukkit.command.CommandSender;
 import org.bukkit.event.Event;
-import org.eclipse.jdt.annotation.Nullable;
-
-import java.util.ArrayList;
-import java.util.List;
+import org.bukkit.event.server.BroadcastMessageEvent;
+import org.jetbrains.annotations.Nullable;
 
 @Name("Broadcast")
 @Description("Broadcasts a message to the server.")
 @Examples({
 	"broadcast \"Welcome %player% to the server!\"",
 	"broadcast \"Woah! It's a message!\""
 })
-@Since("1.0, 2.6 (broadcasting objects), 2.6.1 (using advanced formatting)")
+@Since("1.0, 2.6 (broadcasting objects), 2.6.1 (using advanced formatting), INSERT VERSION (calls broadcast event)")
 public class EffBroadcast extends Effect {
 
+	private static final Pattern HEX_PATTERN = Pattern.compile("(?i)&x((?:&\\p{XDigit}){6})");
+
```

Comment:
Would it be worth file making a public static final for this in `Utils`? we currently have this created in utils, raw string and now effect broadcast.

Reply:
I would love to honestly, but I'm afraid that such changes seems out of scope of this PR for the team. So I'll leave this as is.